dts/bindings/i2c/i2c-device.yaml

```diff
@@ -10,5 +10,6 @@ on-bus: i2c
 properties:
     reg:
       required: true
+      description: device address
```

Comment:
Can you add a bit more here.  Not sure if 'device address' is meaning full enough. 

Maybe `device address on I2C bus`

Reply:
done